examples/forwarding-refs/fancy-button-ref.js

```diff
@@ -3,10 +3,10 @@ import FancyButton from './FancyButton';
 // highlight-next-line
 const ref = React.createRef();
 
-// The FancyButton component we imported is the LogProps HOC.
-// Even though the rendered output will be the same,
-// Our ref will point to LogProps instead of the inner FancyButton component!
-// This means we can't call e.g. ref.current.focus()
+// Komponent FancyButton, który zaimportowaliśmy jest tak naprawdę KWRem LogProps.
```

Comment:
```suggestion
// Komponent FancyButton, który zaimportowaliśmy, jest tak naprawdę KWR-em LogProps.
```

Reply:
Boli to tłumaczenie HOC na KWR może warto dodawać angielskie skróty w nawiasach?